e2e/sidecars/cmd_exec_test.go

```diff
@@ -0,0 +1,36 @@
+package sidecars
+
+import (
+	"context"
+	"strings"
+)
+
+func (s *Suite) TestExecuteCommandInSidecar() {
+	const (
+		namePrefix = "execute-command-in-sidecar"
+		cmdMsg     = "Hello World!"
+		command    = "echo " + cmdMsg
+	)
+
+	target, err := s.Knuu.NewInstance(namePrefix + "-target")
+	s.Require().NoError(err)
+
+	ctx := context.Background()
+	s.Require().NoError(target.Build().SetImage(ctx, alpineImage))
+	s.Require().NoError(target.Build().SetArgs("tail", "-f", "/dev/null")) // Keep the container running
+	s.Require().NoError(target.Build().Commit(ctx))
+
+	sidecar := &testSidecar{
+		StartCommand: []string{"sh", "-c", "sleep infinity"},
+	}
+
+	s.Require().NoError(target.Sidecars().Add(ctx, sidecar))
+	s.Require().NoError(target.Execution().Start(ctx))
```

Comment:
This looks like duplicate code in all the tests that could be refactored into a single helper function.

Reply:
No, all of this code seems duplicate
```
	target, err := s.Knuu.NewInstance(namePrefix + "-target")
	s.Require().NoError(err)

	ctx := context.Background()
	s.Require().NoError(target.Build().SetImage(ctx, alpineImage))
	s.Require().NoError(target.Build().SetArgs("tail", "-f", "/dev/null")) // Keep the container running
	s.Require().NoError(target.Build().Commit(ctx))

	sidecar := &testSidecar{
		StartCommand: []string{"sh", "-c", "sleep infinity"},
	}

	s.Require().NoError(target.Sidecars().Add(ctx, sidecar))
```

Reply:
the start command can be an input to the function. 
all the NewInstance, and commands on `target` seem like they are just copy paste across all these tests.

Reply:
done